### ISSUE TITLE
Scaling HPs for multi-GPU Keras

### ISSUE THREAD

**Issue by ShriyaPalsamudram** [google/automl#932]
I am trying to train efficientdet-D0 on multiple gpus. I am using the code as it is to reach the target accuracy mentioned in the paper. I can train on COCO with 8 Tesla V100 32GB GPUs with the default HPs linearly scaled to get an map of 33.4. But the README and the paper claim an accuracy of 34.3 for APval.
(Note: I am already using synchronized batch normalization as well as soft NMS.)
Additionally, when I try to train on 32 Tesla V100 32GB GPUs by linearly scaling the batch size and HPs, the loss goes to NaN pretty quickly and using basic regularization and HP tuning do not help much.
Can you please share the batch size, learning rate and other HPs used to reproduce the results for D0? Also, are there some scaling restrictions on this model?
Thank you very much.

**Comment by ShriyaPalsamudram** [comment]
@mingxingtan 
I also trained with a global batch size of 128 with HPs as given in the paper. I still got an map of 33.1 for D0.
Is there anything I should do to get the expected 34.3% map.

**Comment by mingxingtan** [comment]
We use the default hyperparamters on TPUs.  Here is the tensorboard for D0:  https://tensorboard.dev/experiment/4IzZoQyAQLSManQfwaHyyQ/#scalars&_smoothingWeight=0
Maybe you can compare this tensorboard with  your tensorboard?
